.github/workflows/e2e-jimm-docker-keycloak.yml

```diff
@@ -73,6 +73,8 @@ jobs:
           timeout_minutes: 1
           max_attempts: 2
           command: while [ "$(docker inspect -f {{.State.Restarting}} jimm)" == "true" ]; do sleep 1; done
+      - name: Remove the JIMM deployed dashboard
+        run: docker container stop juju-dashboard && docker container rm juju-dashboard
```

Comment:
@kian99 Would it be possible to have jimm's docker-compose ([here](https://github.com/canonical/jimm/pull/1684/files#diff-3fde9d1a396e140fefc7676e1bd237d67b6864552b6f45af1ebcc27bcd0bb6e9R142)) be able to deploy from a local docker image using an env var or something? That would mean we could just build the image and let jimm handle the rest.